README.md

```diff
@@ -39,45 +39,48 @@ When you finish the instructions below, you will have the beginning of a best pr
  - Optimal tuning of PE settings for this configuration
  - Working and example [roles and profiles](https://docs.puppetlabs.com/pe/2015.3/puppet_assign_configurations.html#assigning-configuration-data-with-role-and-profile-modules) code
 
-# How to set it all up
+# Setup
 
-## Copy this repo into your own Git server
+## Copy this repo to your Git server
 
 ### GitLab
 
-1. On a new server, install GitLab
+1. Install GitLab.
  - https://about.gitlab.com/downloads/
 
-2. After GitLab is installed you may sign into the web UI with the `root` user and password `5iveL!fe`
+2. Navigate to your GitLab server's FQDN. Log into GitLab as the user `root`.
 
-3. In the GitLab UI, make a user for yourself
+3. In the GitLab UI, create a user to edit code with.
 
-4. From your laptop or development machine, make an SSH key to link with your user.
- - Note: this is used for your laptop to communicate with your GitLab server to push code
- - https://help.github.com/articles/generating-ssh-keys/
- - http://doc.gitlab.com/ce/ssh/README.html
+4. From your laptop or development machine, generate an SSH key to link with your user. This key allows your laptop to communicate with your GitLab server and push code.
 
-5. In the GitLab UI, create a group called `puppet` ( this is case sensitive )
+ - More on generating SSH keys:
+      - https://help.github.com/articles/generating-ssh-keys/
+      - http://doc.gitlab.com/ce/ssh/README.html
+
+5. In the GitLab UI, create a group called `puppet`.
  - http://doc.gitlab.com/ce/workflow/groups.html
 
-7. In the GitLab UI, add your user to the `puppet` group
- - Make sure to give your user at least master permissions so you can complete the below steps
- - Read more about permissions:
+6. Add your user to the `puppet` group.
+ - You must give your user at least master permissions to complete these steps.
+ - Read more about GitLab permissions:
     - https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/permissions/permissions.md
 
-8. In the GitLab UI, create a project called `control-repo` and set the Namespace to be the `puppet` group
+7. Create a project called `control-repo` and set its Namespace to `puppet`.
+
+  - When *control-repo* is created, GitLab displays its HTTP URL. To the left of the URL field, select **SSH** to view the SSH URL. The SSH URL for this GitLab project (repo) is used to configure your local control-repo and PE, take note of it.
 
-10. On your laptop, clone this GitHub control repo
- - `git clone <repo URL>`
+8. On your laptop, clone this repo.
```

Comment:
I think it would be good to have steps 1-8 as 'Setup a GitLab server' heading and the 'on your laptop steps' as another heading called 'Clone control-repo', especially if you're going to add other Git providers. I didn't want to overload changes though.